src/util/image/PlacementPicker.java

```diff
@@ -373,7 +373,7 @@ public void actionPerformed(final ActionEvent event) {
     menuBar.add(fileMenu);
     menuBar.add(editMenu);
   }// end
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 // constructor
+   // constructor
```

Comment:
Please do not add comments to indicate class structure. They've more a maintenance burden than anything else, very likely to become out dated and incorrect as the code changes.